### ISSUE TITLE
Release 0.12?

### ISSUE THREAD

**Issue by caspervdw** [pygeos/pygeos#429]
I think it is a good time for a release, there are a few major enhancements that I would like to publish.

@jorisvandenbossche This might be a good moment also to move the active work to the shapely repo?

**Comment by caspervdw** [comment]
Or maybe it is good to start adding some deprecation warnings (#398 for instance?)

**Comment by caspervdw** [comment]
The failing conda forge build (https://github.com/conda-forge/pygeos-feedstock/pull/48) gives the release some urgency.

**Comment by jorisvandenbossche** [comment]
It's not super urgent for conda-forge, since it would also be easy to skip that specific test. But I certainly agree on doing a release!

**Comment by jorisvandenbossche** [comment]
OK, I think we have everything merged we want for 0.12? 

Assuming that, I am going to merge the history into Shapely today.

**Comment by caspervdw** [comment]
One more: #445

**Comment by caspervdw** [comment]
Release 0.12 has been done. Everything except the aarch64 builds are now on PyPI. The aarch64 builds keep failing in Travis: https://app.travis-ci.com/github/pygeos/pygeos/jobs/550790614 . I just rebooted it for the third time, hope it works now.

**Comment by caspervdw** [comment]
It seems that Travis didn’t trigger on our tag, so we won’t have aarch64 builds until we fix that.

**Comment by caspervdw** [comment]
I POSTed the following request to the Travis API v3:

url: `https://api.travis-ci.com/repo/12929828/requests`
```
{
    "request": {
        "branch": "0.12.0",
        "config": {
            "branches": {
                "only": [
                    "0.12.0"
                ]
            }
        }
    }
}
```

Which seems to have triggered the build: https://app.travis-ci.com/github/pygeos/pygeos/builds/242890059

**Comment by caspervdw** [comment]
@odidev I just released 0.12 with a new build CI for the aarch64 wheels. Do you have the time to test one of them?

**Comment by jorisvandenbossche** [comment]
Thanks @caspervdw !

**Comment by odidev** [comment]
> @odidev I just released 0.12 with a new build CI for the aarch64 wheels. Do you have the time to test one of them?

@caspervdw I have tested and it is working fine.
